Packages/Tracking Preview/PhysicsHands/Runtime/PhysicsHand.cs

```diff
@@ -459,6 +459,14 @@ public void UpdateHand(Leap.Hand hand)
             PhysicsHandsUtils.ConvertPhysicsToLeapHand(_physicsHand, ref _leapHand, _originalLeapHand, Time.fixedDeltaTime);
 
             OnUpdatePhysics?.Invoke();
+
+            CacheOldValues();
+        }
+
+        private void CacheOldValues()
```

Comment:
I think this would be good to be called something more specific, so when reading the code that calls it, you don't need to work out what is being cached?

like - CachePositionsForVelocity

Reply:
I couldn't think of a name that was more specific as I've included the `_originalOldPosition` caching here too, which is used to determine positional delta issues that could arise from hand teleportation

Reply:
Could it just be 'CachePositions'?

I think it's the word Values that scares me